packages/vsce/src/doc/meta/JVMServer.meta.ts

```diff
@@ -0,0 +1,63 @@
+import { CicsCmciConstants } from "@zowe/cics-for-zowe-sdk";
+import { Resource } from "../../resources/Resource";
+import { PersistentStorage } from "../../utils/PersistentStorage";
+import { IResourceMeta } from "./IResourceMeta";
+import { IJVMServer } from "../resources/IJVMServer";
+
+const persistentStorage = new PersistentStorage("zowe.cics.persistent");
+
+export const JVMServerMeta: IResourceMeta<IJVMServer> = {
+  resourceName: CicsCmciConstants.CICS_JVMSERVER_RESOURCE,
+  humanReadableName: "JVM Servers",
+
+  buildCriteria(criteria: string[]) {
+    return criteria.map((n) => `NAME=${n}`).join(" OR ");
+  },
+
+  async getDefaultCriteria() {
+    return PersistentStorage.getDefaultFilter(CicsCmciConstants.CICS_JVMSERVER_RESOURCE, "jvmServer");
+  },
+
+  getLabel: function (jvmServer: Resource<IJVMServer>): string {
+    let label = `${jvmServer.attributes.name}`;
+    if (jvmServer.attributes.enableStatus && jvmServer.attributes.enableStatus.toLowerCase() === "disabled") {
+      label += " (disabled)";
+    }
+    return label;
+  },
+
+  getContext: function (jvmServer: Resource<IJVMServer>): string {
+    const status = (jvmServer.attributes.enableStatus ?? "").trim().toUpperCase();
+    return `${CicsCmciConstants.CICS_JVMSERVER_RESOURCE}.${status}.${jvmServer.attributes.name}`;
+  },
+
+  getIconName: function (jvmServer: Resource<IJVMServer>): string {
+    let iconName = `jvm-server`;
+    const status = (jvmServer.attributes.enableStatus ?? "").trim().toUpperCase();
+    if (status === "DISABLED") {
+      iconName += `-disabled`;
+    }
+    return iconName;
+  },
+
+  getHighlights(jvmServer: Resource<IJVMServer>) {
+    return [
+      {
+        key: "Status",
+        value: jvmServer.attributes.enableStatus,
+      },
```

Comment:
I reckon we could add profile here too, as a second highlighted attribute. More to come in the future I'm sure, but this is a good thing to highlight for now

```suggestion
      },
      {
        key: "Profile",
        value: jvmServer.attributes.profile,
      }
```

Reply:
Resolved